auth-demo/src/main/resources/application.yml

```
@@ -3,3 +3,4 @@ server:
 vaadin:
   auth:
     auto-configure: true
+    logout-redirect-url: /
```

It should be `logout-redirect-route`, according to the getters/setters at `VaadinAuthProperties` class.
```suggestion
    logout-redirect-route: /
```